config/lessons.exs

```diff
@@ -52,7 +52,7 @@ config :school_house,
       :basics,
       :changesets,
       :associations,
-      :query_basics,
+      :querying_basics,
```

Comment:
I need to dig in but why does this need to change but not the one below?

Reply:
I just looked around a little, and I don't see a reference to `query_advanced`, `querying_advanced`, or `Querying: Advanced` in the school_house or elixirschool repos. Am I missing it somewhere or is it a TBD lesson idea?

Reply:
TBD. Let me review this myself before we merge just to see if I made some mistakes I should change.